example-definitions/cli-target-grpc.yaml

```diff
@@ -1,3 +1,4 @@
+auth_mode: "basic"
```

Comment:
minor nit, you have camel case attributes like restURL and this is using underscores, so maybe it should be authMode?